pkg/cmd/render/render.go

```diff
@@ -45,17 +45,27 @@ func NewRenderCommand(errOut io.Writer) *cobra.Command {
 		Use:   "render",
 		Short: "Render kubernetes controller manager bootstrap manifests, secrets and configMaps",
 		Run: func(cmd *cobra.Command, args []string) {
-			must := func(fn func() error) {
+			must := func(fn func() error) error {
```

Comment:
the whole point of `must` is that it doesn't return an error.

Reply:
Yes, I know, but it makes it un-unit-testable.

Normally the code flow would call `klog.Fatalf`, but that would kill the test. But it means the unit tests cannot be representative when an error comes out of any of the three stages of `must`.

In this case, one of the tests checks what happens when there are multiple, contradictory feature gates, the validate returns an error, but the rest of the steps continue uninterrupted, producing an incorrect output